extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js

```diff
@@ -14,14 +14,34 @@
  * limitations under the License.
  */
 
-// import {dev} from '#utils/log';
-
 import {AmpAccessFewcents} from '../fewcents-impl';
 
 const TAG = 'amp-access-fewcents';
 
 const TAG_SHORTHAND = 'aaf';
 
+const paywallResponse = {
+  success: true,
+  message: 'Amp article price returned with unlock url.',
+  data: {
+    loginUrl:
+      'https://wallet.hounds.fewcents.co?login_source=amp_paywall&login_publisher_id=47&login_publisher_bid_id=90531&bid_mode=paywall&bid_price=622.38&login_redirect_url=http%3A%2F%2Flocalhost%3A8000%2Fexamples%2Famp-access-fewcents.html&login_publisher_logo_url=https%3A%2F%2Fi1.wp.com%2Fthefix.media%2Fwp-content%2Fuploads%2F2021%2F04%2Flogo-no-margins.png&amp_reader_id=amp-yfpD7ewsxdSdbH4xCboNIw',
```

Comment:
Should the URL be `https://wallet.hounds.fewcents.co/?xxxx`? Also on 2 other occasions in this PR. The backslash should be required in URL. Maybe many browsers can work it fine, but still better to follow the standard.

Reply:
Fixed.